src/routes/docs/quick-starts/android-java/+page.markdoc

```diff
@@ -0,0 +1,393 @@
+---
+layout: article
+title: Start with Android (Java)
+description: Get started with Appwrite on Android using Java and learn how to build secure and scalable apps using our powerful backend.
+difficulty: beginner
+readtime: 3
+back: /docs/quick-starts
+---
+
+Learn how to setup your first Android project powered by Appwrite and the [Appwrite Android SDK](https://github.com/appwrite/sdk-for-android) using Java.
+
+{% info title="Using Kotlin?" %}
+Check out the [Start with Android (Kotlin)](/docs/quick-starts/android) guide.
+{% /info %}
+
+{% section #step-1 step=1 title="Create Android project" %}
+Open Android Studio and click **New Project** to create a new project.
+
+Choose your desired project template, for example **Empty Activity**, and click **Next**.
+
+Now enter your app **name** and **package name**. You will need both of these later when you create your project in the Appwrite console. Click **Finish** to create your project.
+
+{% /section %}
+
+{% section #step-2 step=2 title="Create Appwrite project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console).
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+If this is your first time using Appwrite, create an account and create your first project.
+
+Then, under **Add a platform**, add an **Android app**.
+
+Add your app's **name** and **package name**, your package name is the one entered when creating an Android project. For existing projects, you should use the **applicationId** in your app-level [build.gradle](https://github.com/appwrite/playground-for-android/blob/master/app/build.gradle#L11) file.
+
+{% only_dark %}
+![Add a platform](/images/docs/quick-starts/dark/add-platform.png)
+{% /only_dark %}
+{% only_light %}
+![Add a platform](/images/docs/quick-starts/add-platform.png)
+{% /only_light %}
+
+You can skip optional steps.
+
+{% /section %}
+
+{% section #step-3 step=3 title="Add the Appwrite SDK" %}
+To add the Appwrite SDK for Android as a dependency, add the following to your app-level **build.gradle** file inside the **dependencies** block.
+
+```java
+implementation "io.appwrite:sdk-for-android:7.0.0"
+```
+
+In order to allow creating OAuth sessions, the following activity needs to be added inside the `<application>` tag, along side the existing `<activity>` tags in your [AndroidManifest.xml](https://github.com/appwrite/playground-for-flutter/blob/master/android/app/src/main/AndroidManifest.xml).
+Be sure to replace the **[PROJECT_ID]** string with your actual Appwrite project ID.
```

Comment:
Let's be consistent with the format of our placeholders. Please check the rest of the page as well.

```suggestion
Be sure to replace the **<PROJECT_ID>** string with your actual Appwrite project ID.
```

Reply:
Got it, @stnguyen90. I used the same format as the current Android Kotlin quick start. Should I update that too?

Reply:
@stnguyen90 I've updated this one. Looking at it though, I think the square brackets were used to not confuse it with the tags.

Reply:
@stnguyen90 Here's an example of what we have now:

`<data android:scheme="appwrite-callback-<PROJECT_ID>" />`

Reply:
@ebenezerdon we used to use square brackets. We've switched to angle brackets but there's still a lot of old content that still uses the square brackets.

Reply:
Got it. Will do 👌